frontend/src/core/i18n/nl.json

```
@@ -7,6 +7,7 @@
     "settings": "Instellingen"
   },
   "settings": {
+    "actions": { "delete": "Verwijderen", "cancel": "Annuleren" },
```

Biome (2.4.15)

[error] The key actions was already declared.

(lint/suspicious/noDuplicateObjectKeys)